src/RestApiClient.js

```diff
@@ -74,7 +74,8 @@ class RestApiClient {
    * @return {String}
    */
   getLocalUrl() {
-    return `//${window.location.hostname}${window.location.port}`;
+    const port = window.location.port ? `:${window.location.port}` : '';
```

Comment:
that's weird. I had checked this and `window.location.port` would default to empty string already if nothing was assigned to it...